### ISSUE TITLE
cosmos theme - blockquotes

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#3276]
![Screen Shot 2020-12-21 at 6 06 21 PM](https://user-images.githubusercontent.com/11299125/102833643-46277800-43b7-11eb-97de-ea69f99066f4.png)

Same font as surrounding text, margins on left and right are 0.25 inches in addition to the standard 1inch

**Comment by ajluk** [comment]
@lwtchu This how it looks: 
![image](https://user-images.githubusercontent.com/55203598/105375996-90478780-5c09-11eb-9817-3f4b71632e72.png)
I have noticed accidentally that there also blockquotes in Sociology. Do you want to open a separate card for this book or I can do it? I have added the blockquotes style for this book as well within the same PR.

**Comment by ajluk** [comment]
@lwtchu I have made an investigation about `blockquotes`, because me and Natalia have noticed that the blockquotes have default properties vaules as an html tag and Prince supports and uses them. There is no difference between styled and unstyled blockquote in PDF's. So I'm wondering if we should add the styles in the code just in case Prince will no longer supports the defaults (as it was done before)? Or should we reset the default values and only set the margins in the code? I have checked if I could reset them and in case of left and right margins yes, but top and bottom can still be seen in the pdf after reset. 
Here are the default values: 
 ```
 display: block;
  margin-top: 1em;
  margin-bottom: 1em;
  margin-left: 40px;
  margin-right: 40px;
```
What is you opinion?

**Comment by lwtchu** [comment]
@ajluk could you explain the margins part? i'm not sure i quite follow 😓

**Comment by ajluk** [comment]
@lwtchu Sure! Maybe the best way could be to explain it on the example of browser default properties values for blockquotes (this is how it looks in the html when I'm resetting them): 
![image](https://user-images.githubusercontent.com/55203598/105522127-24ceea00-5cdd-11eb-972b-f4fb75d3849e.png)

**Comment by lwtchu** [comment]
@ajluk hmm if it's not too much trouble i think we should add the styles in the code in case it's no longer supported or if the default changes since we have specific values we already want.

does that make sense D:

**Comment by ajluk** [comment]
Here is the blockquote in the PDF file with reset margin values: 
![image](https://user-images.githubusercontent.com/55203598/105523818-2d282480-5cdf-11eb-90da-237784eaabe8.png)

**Comment by ajluk** [comment]
@lwtchu Yes! :relaxed: So to add the values we exactly want to left/right margins (0.25in) I need reset them first in the reset.scss file and add the values via code. Is that ok? It will affect all themes, but as far as I know only in `cardboard` we had blockquotes. Should Marta then test one book form `cardboard` just to check? Maybe I will create separate issue for cardboard?

**Comment by lwtchu** [comment]
@ajluk that's okay. it'll be a good idea to test a book from cardboard. whenever it goes thru you can make the card?

**Comment by ajluk** [comment]
Oh, thank you! I will make the card for cardboard.

**Comment by ajluk** [comment]
Here is the card for cardboard theme:  #3409 (I'm pasting it here just to connect).
